src/ast/mod.rs

```diff
@@ -431,6 +431,12 @@ impl fmt::Display for WindowFrameBound {
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum Statement {
+    //
```

Comment:
I think you want to add something like `/// EXPLAIN`  here?

Reply:
Added, thanks